ansible/inventories/local_development/group_vars/all

```diff
@@ -7,16 +7,20 @@ dataserver_1_internal_ip: 127.0.0.1
 deployment_environment_id: local_development
 ssl_certificate_name: nginx-selfsigned.crt
 ssl_key_name: nginx-selfsigned.key
+redirect_ssl_certificate_name: redirect-nginx-selfsigned.crt
+redirect_ssl_certificate_key_name: redirect-nginx-selfsigned.key
 web_app_github_repo_branch: test
 search_app_github_repo_branch: test
 server_domain_name: etsin-finder.local
+redirect_server_domain_name: redirect.etsin-finder.local
 
 nginx_es_credentials:
   - { username: etsin, password: test-etsin }
 
 gunicorn_preload: false
 
 ssl_certificates_path: /etc/nginx/ssl_certs
+redirect_ssl_certificates_path: /etc/nginx/ssl_certs
```

Comment:
I think this variable is unnecessary. The existing ``ssl_certificates_path`` directory should be ok for these certs as well.

Reply:
Actually looks like you did use ``ssl_certificates_path``, but probably just forgot this variable here.

Reply:
Yes, you are correct, this variable is unnecessary. It was created to account for a separate certificate location for the old production certificate, but since the variable isn't used in production, it's not needed.